celavi/pylca_celavi/pylca_opt_foreground.py

```diff
@@ -231,15 +188,15 @@ def runner(tech_matrix,F,yr,i,j,k,route_id,state,final_demand_scaler,process,df_
        return res
     
     else:        
-       print(f"optimization pylca-opt-foreground emission failed  for {k} at {j} in {yr}")
+       print(f"pylca-opt-foreground emission failed  for {k} at {j} in {yr}")
 
 
 def model_celavi_lci(f_d,yr,fac_id,stage,material,route_id,state,df_static,dynamic_lci_filename,electricity_grid_spatial_level,intermediate_demand_filename):
 
     """
-    Main function of this module which received information from DES interface and runs the suppoeting optimization functions. 
+    Main function of this module which received information from DES interface and runs the supporting LCA functions. 
     Creates the technology matrix and the final demand vector based on input data. 
-    Performs necessary checks before and after the LCA optimization calculation. 
+    Performs necessary checks before and after the LCA calculation. 
```

Comment:
More detail here would be helpful - what checks are performed, what errors do they prevent?

Reply:
Added detailed information